### ISSUE TITLE
Connecting to new SaaS instance option results in error when clicking Save & test button

### ISSUE THREAD

**Issue by jHumeLPL** [prydin/aria-operations-plug-in-for-grafana#33]
When choosing the new "Connecting to SaaS instance?" check box and leaving the "SaaS Region" text box blank (because we use the US region), you get an error of "**object [Object] is not valid JSON**".

However, if you do go ahead and just put in "us" in the "SaaS Region" box, it works.  So, no big deal really, just thought I should let you know that leaving it blank doesn't seem to work.

Also, even though the text box says "SaaS Region", the text in the box itself still shows "Aria Operations Host or IP" which may be a bit confusing for some.  Again, not a big deal at all.  Just thought you may want to know.

Anyways, again, this is really great.  Much appreciated!

**Comment by prydin** [comment]
Are you sure that your API key is correct? This error seems to happen when the key is incorrect. I'll fix the error message, but the underlying cause seems to be the authentication failing.

**Comment by jHumeLPL** [comment]
Yeah, the API key is correct.  Like I said, if I type in "us" in the SaaS region box, it works fine.  

As a matter of fact, what's weird is, after I typed it in and saved it, then go back to it and empty that box back out, it will still work.  But, until I had typed it in and saved it at least once, it kept giving me that error.  Very weird.

I just retested the issue by deleting the data source and re-configuring it and yep, same issue.  Leaving it blank results in that message until I manually type "us".

I attached screenshots to make it more clear what I'm talking about.  Again, this is _very_ much a non-issue, but then again, it could be confusing for some, as it did trip me up for a bit.  I was typing up an issue saying that it didn't work at all when I finally had the idea to just manually type "us" in the region box even though it says to leave it blank for US region.

![2024-01-03_11-20-10](https://github.com/prydin/aria-operations-plug-in-for-grafana/assets/76532989/93894549-4240-4762-9935-9cc648d8402c)
